From e974860777f45ddc09f337ec69ee9c0068469a72 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 16 Jan 2015 13:57:47 +0800
Subject: [PATCH] cleanup: android argument for sama5d4 boards

When porting android for sama5d4 related boards, they are no
longer supporting boot up android through bootstrap directly,
so remove android arguments for sama5d4 based boards.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 Config.in                                    | 2 --
 board/sama5d4_xplained/Config.in.android_arg | 3 ---
 board/sama5d4ek/Config.in.android_arg        | 3 ---
 3 files changed, 8 deletions(-)
 delete mode 100644 board/sama5d4_xplained/Config.in.android_arg
 delete mode 100644 board/sama5d4ek/Config.in.android_arg

diff --git a/Config.in b/Config.in
index 28d42fc..df8bf6e 100644
--- a/Config.in
+++ b/Config.in
@@ -167,8 +167,6 @@ config CONFIG_MEM_SIZE
 config CONFIG_LINUX_KERNEL_ARG_STRING
 	string "android kernel command line"
 source "board/sama5d3xek/Config.in.android_arg"
-source "board/sama5d4ek/Config.in.android_arg"
-source "board/sama5d4_xplained/Config.in.android_arg"
 
 config CONFIG_IMG_ADDRESS
 	depends on CONFIG_DATAFLASH || CONFIG_FLASH || CONFIG_NANDFLASH
diff --git a/board/sama5d4_xplained/Config.in.android_arg b/board/sama5d4_xplained/Config.in.android_arg
deleted file mode 100644
index a43c4b8..0000000
--- a/board/sama5d4_xplained/Config.in.android_arg
+++ /dev/null
@@ -1,3 +0,0 @@
-config CONFIG_LINUX_KERNEL_ARG_STRING
-	default "console=ttyS0,115200 mtdparts=atmel_nand:5M(Bootstrap),125M(system),-(userdata) ubi.mtd=1 ubi.mtd=2 rw root=ubi0:system rootfstype=ubifs init=/init androidboot.hardware=sama5d4_xplained androidboot.console=ttyS0" if CONFIG_SAMA5D4_XPLAINED && !CONFIG_SDCARD
-	default "console=ttyS0,115200 root=/dev/mmcblk0p2 rw rootwait init=/init androidboot.hardware=sama5d4_xplained androidboot.console=ttyS0" if CONFIG_SAMA5D4_XPLAINED && CONFIG_SDCARD
diff --git a/board/sama5d4ek/Config.in.android_arg b/board/sama5d4ek/Config.in.android_arg
deleted file mode 100644
index 754e818..0000000
--- a/board/sama5d4ek/Config.in.android_arg
+++ /dev/null
@@ -1,3 +0,0 @@
-config CONFIG_LINUX_KERNEL_ARG_STRING
-	default "console=ttyS0,115200 mtdparts=atmel_nand:5M(Bootstrap),125M(system),-(userdata) ubi.mtd=1 ubi.mtd=2 rw root=ubi0:system rootfstype=ubifs init=/init androidboot.hardware=sama5d4x-ek androidboot.console=ttyS0" if CONFIG_SAMA5D4EK && !CONFIG_SDCARD
-	default "console=ttyS0,115200 root=/dev/mmcblk0p2 rw rootwait init=/init androidboot.hardware=sama5d4x-ek androidboot.console=ttyS0" if CONFIG_SAMA5D4EK && CONFIG_SDCARD
-- 
1.9.1

